src/routes/Settings/useProfileSettingsInputs.js

```diff
@@ -213,15 +213,15 @@ const useProfileSettingsInputs = (profile) => {
         }
     }), [profile.settings]);
     const playInExternalPlayerCheckbox = React.useMemo(() => ({
-        checked: profile.settings.playInExternalPlayer,
+        checked: profile.settings.playerType === 'external',
         onClick: () => {
             core.transport.dispatch({
                 action: 'Ctx',
                 args: {
                     action: 'UpdateSettings',
                     args: {
                         ...profile.settings,
-                        playInExternalPlayer: !profile.settings.playInExternalPlayer
+                        playerType: profile.settings.playerType !== 'external' ? 'external' : 'internal'
```

Comment:
I don't have the whole context here, but possible typo?

```suggestion
                        playerType: profile.settings.playerType === 'external' ? 'external' : 'internal'
```

Reply:
never mind I commented too soon. this is probably supposed to be flipped.

Reply:
it needs to update the value to the opposite of the current value, so the code is already correct, i think